conf/nonEeaBusiness.routes

```diff
@@ -24,6 +24,9 @@ POST       /check-answers                                             controller
 GET        /:index/remove                                             controllers.asset.noneeabusiness.remove.RemoveAssetYesNoController.onPageLoad(index: Int)
 POST       /:index/remove                                             controllers.asset.noneeabusiness.remove.RemoveAssetYesNoController.onSubmit(index: Int)
 
+GET        /:index/remove-with-end-date                               controllers.asset.noneeabusiness.remove.RemoveAssetEndDateController.onPageLoad(index: Int)
```

Comment:
Could this endpoint be named /:index/remove/end-date to keep aligned with other removals?

Reply:
Done